.circleci/config.yml

```diff
@@ -1,7 +1,76 @@
-version: 2
+version: '2.1'
+orbs:
+  gh: circleci/github-cli@1.0.4
 
 jobs:
-  build:
+  generate-docs:
+    docker:
+      - image: circleci/python:3.7.9-stretch
+
+    steps:
+      - checkout
+
+      - run:
+          run: setup_creds
+          command: |
+            echo $GCLOUD_SERVICE_KEY | base64 --decode --ignore-garbage > ${HOME}/gcloud-service-key.json
+
+      - add_ssh_keys:
+          fingerprints:
+            - "0a:34:dc:0a:7d:86:e9:b1:e2:a8:9d:af:ef:6d:c2:9c"
```

Comment:
Where is this key being added from?

Reply:
From what I understand, CircleCI needs an ssh key to communicate with github. We create this key locally using `ssh-keygen`. In CircleCI, we add the id_rsa key to the project & then in GH we add the id_rsa.pub -- this will allow CircleCI to read/write to GH. After you add the id_rsa key to CircleCI it gives a fingerprint ID like the one seen above.

Reply:
That makes a lot of sense and I believe you are right regarding what this is. Thanks!